src/open_inwoner/conf/base.py

```diff
@@ -123,6 +123,9 @@
 CACHE_ZGW_CATALOGI_TIMEOUT = config("CACHE_ZGW_CATALOGI_TIMEOUT", default=60 * 60 * 24)
 CACHE_ZGW_ZAKEN_TIMEOUT = config("CACHE_ZGW_ZAKEN_TIMEOUT", default=60 * 1)
 
+# Maximum number of pagination requests to follow when fetching zaken from ZGW APIs
+ZGW_MAX_REQUESTS = config("ZGW_MAX_REQUESTS", default=4)
```

Comment:
```suggestion
ZGW_MAX_REQUESTS = config("ZGW_MAX_REQUESTS", default=8)
```

I think we'll need a higher max as well: Zwolle was testing cases with ~120 cases. A higher limit will only be an issue for users that actually have that many pages, which I think is acceptable.